server/server.ts

```diff
@@ -0,0 +1,113 @@
+import { Request, Response } from "express";
+import { getTokenxToken } from "./auth/tokenx";
+
+require("dotenv").config();
+const express = require("express");
+const path = require("path");
+const getHtmlWithDecorator = require("./dekorator");
+const logger = require("./logger");
+const decodeJWT = require("jwt-decode");
+const cookies = require("cookie-parser");
+const { createProxyMiddleware } = require("http-proxy-middleware");
+const compression = require("compression");
+const { getAzureadToken } = require("./auth/azuread");
+const fetch = require("node-fetch");
+
+const server = express();
+const buildPath = path.resolve(__dirname, "../../build");
+const baseUrl = "/person/kontakt-oss/tilbakemeldinger";
+const validPaths = ["ros", "serviceklage", "feil-og-mangler"];
+
+const getAccessToken = async (req: Request) => {
+  const selvbetjeningIdToken = req.cookies["selvbetjening-idtoken"];
+
+  if (req.params.path === "serviceklage" && selvbetjeningIdToken) {
+    try {
+      return await getTokenxToken(
+        selvbetjeningIdToken,
+        `${process.env.ENV}-gcp:teamserviceklage:tilbakemeldingsmottak-api`
+      );
+    } catch (e) {
+      console.log(
+        "Failed to fetch tokenx token, fetching Azure AD token as fallback"
+      );
+    }
+  }
+
+  return await getAzureadToken(
+    `api://${process.env.ENV}-gcp.teamserviceklage.tilbakemeldingsmottak-api/.default`
+  );
+};
+
+// Parse application/json
+server.use(compression());
+server.use(baseUrl, express.static(buildPath, { index: false }));
+server.use(cookies());
+server.use(express.json());
+server.get(
+  `${baseUrl}/internal/isAlive|isReady`,
+  (req: Request, res: Response) => res.sendStatus(200)
+);
+
+server.get(`${baseUrl}/fodselsnr`, (req: Request, res: Response) =>
+  res.send({ fodselsnr: decodeJWT(req.cookies["selvbetjening-idtoken"]).pid })
+);
+
+server.post(`${baseUrl}/mottak/:path`, async (req: Request, res: Response) => {
```

Comment:
Kan erstatte dette med noe sånt, så vil denne routen kun svare på de ønskede path'ene. Kan da fjerne sjekk av ugyldige paths.
```suggestion
server.post(`${baseUrl}/mottak/:path(ros|serviceklage|feil-og-mangler)`, async (req: Request, res: Response) => {
```

Reply:
Fikset